libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/client/Sleeper.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.di.ipv.core.library.client;
+
+public interface Sleeper {
+    public void sleep(long millis) throws InterruptedException;
```

Comment:
(Given that you're mocking this out anyway, this could just be a real implementation)